django/contrib/postgres/fields/array.py

```diff
@@ -179,6 +181,19 @@ def as_sql(self, qn, connection):
 
 
 @ArrayField.register_lookup
+class ArrayIn(In):
+    def batch_process_rhs(self, compiler, connection, rhs=None):
+        sqls, sqls_params = [], []
+        for p in rhs:
+            value = QueryWrapper(
```

Comment:
I don't see any docs for `QueryWrapper` --  could it be useful to document it now or later?